packages/client/src/app/components/site-sidenav/notifications/notif-item/notif-item.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { slugify } from 'voca';
+import { Store } from '@ngxs/store';
+import { Navigate } from '@ngxs/router-plugin';
+
+import { ContentKind } from '@pulp-fiction/models/content';
+
+@Component({
+    selector: 'notif-item',
+    templateUrl: './notif-item.component.html',
+    styleUrls: ['./notif-item.component.less']
+})
+export class NotifItemComponent implements OnInit {
+    @Input() notif: any; // this is `any` because different notification documents have different fields
```

Comment:
This feels like a sign that I need to make the interfaces inherit from the NotificationBase interface. And I think I can even do that...

Reply:
i'll have to work on this in a future update, as currently it breaks a few things